### ISSUE TITLE
100 songs limit on artist pages with more than 100 songs

### ISSUE THREAD

**Issue by assier08** [Malopieds/InnerTune#234]
### Checklist

- [X] I am able to reproduce the bug with the [latest debug version](https://github.com/Malopieds/InnerTune/actions).
- [X] I've checked that there is no open or closed issue about this bug.
- [X] This issue contains only one bug.
- [X] The title of this issue accurately describes the bug.

### Steps to reproduce the bug

Open app 
Go to an artist pages (make sure that it has more than  100 songs, approximately)
After that number of songs, the app won't show any song after the 100th

### Expected behavior

All songs should load correctly

### Actual behavior

After the 100th song, any song won't be loaded

### Screenshots/Screen recordings

I don't know if i'ts a real bug or the app has a  limit of songs, because I had the same problem with apps like RiMusic, but, just in case here is a comparison with YouTube Music, with the band "zebrahead"

https://github.com/user-attachments/assets/6fec6f46-6a91-49eb-9707-502113071b93

### Logs

   ︎︎

### InnerTune version

0.5.8

### Android version

Android 14

### Additional information

As I said before, I don't know if It is a bug or not, but there you have the comparation between innertune and Yt music.

**Comment by ghost** [comment]
experiencing the same with playlists

**Comment by Richard38907** [comment]
Duplicate of #136 I think🤔. Same problem here.

**Comment by Malopieds** [comment]
Yeah it's kind of a duplicate, because it's youtube only sending the first 100. I will keep this one open to remember to test it though!

**Comment by Hikari2w2** [comment]
Same problem with playlists ☹️

**Comment by Gamr13** [comment]
Seems like z-huang's InnerTune isn't impacted by this bug, it just continuously loads until the end, might be worth comparing how both apps load their playlists or pull info from YouTube Music.

I'd love to see this bug fixed, this fork is very much my favourite of all the clients!

**Comment by Gamr13** [comment]
I've noticed if you press a song in your likes playlist, there's a perpetual list of songs to play next that can pull past the 100 song limit, but the actual playlist cannot. 

Awkward bypass, but it "works"

**Comment by Carter0108** [comment]
Same bug effects albums with more than 100 tracks. Only the first 100 show.

Example album that has 110 tracks:

https://music.youtube.com/browse/MPREb_Zk6wFf5XJyi

**Comment by AsterionXeity** [comment]
For me the whole playlist doesn't load if it has over 100 songs in it

**Comment by paypay-komi** [comment]
I'm experiencing the same issue with my liked music playlist and other large playlists; they load indefinitely.  However, I've found that it works correctly on Outertune.  I prefer this fork, though, because it has Discord RCP.
